client/state/geo/selectors.js

```diff
@@ -32,3 +32,13 @@ export function getGeo( state ) {
 export function getGeoCountry( state ) {
 	return get( getGeo( state ), 'country_long', null );
 }
+
+/**
+ * Returns the current browser IP geolocation abbreviated country name.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}       Current browser IP geolocation short country name
+ */
+export function getGeoCountryShort( state ) {
```

Comment:
Should I move all the others in the file then too? The only reason I added it there was the rest that it relates to are there.